src/frontend/components/Dashboard.jsx

```diff
@@ -17,7 +18,49 @@ import FormTab from './dashboard/FormTab.jsx';
 import MapTab from './dashboard/MapTab.jsx';
 import SettingsTab from './dashboard/SettingsTab.jsx';
 
-export default function NavTabs() {
+function UserTabs(props) {
+  const { user, inputs, handleSettings } = props;
+  console.log('user: ', user);
+  if (user === 'admin') {
```

Comment:
Is this actually checking the username or is the checking the role? It's a little unclear, just wanted to check. Should definitely be checking the role since the username can be changed.

Reply:
It's checking the role, but only locally and not through the oauth yet. Adding a commit to pluralize these to reflect the syntax I believe groups will have from the saas verison.